src/server/scripts/Commands/cs_event.cpp

```diff
@@ -80,28 +81,18 @@ class event_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleEventInfoCommand(ChatHandler* handler, char const* args)
+    static bool HandleEventInfoCommand(ChatHandler* handler, Variant<Hyperlink<gameevent>, uint32> const eventId)
     {
-        if (!*args)
-            return false;
-
-        // id or [name] Shift-click form |color|Hgameevent:id|h[name]|h|r
-        char* id =  handler->extractKeyFromLink((char*)args, "Hgameevent");
-        if (!id)
-            return false;
-
-        uint32 eventId = atoul(id);
-
         GameEventMgr::GameEventDataMap const& events = sGameEventMgr->GetEventMap();
 
-        if (eventId >= events.size())
+        if (*eventId >= events.size())
```

Comment:
same question

Reply:
```
../src/server/scripts/Commands/cs_event.cpp:88:21: error: no match for ‘operator>=’ (operand types are ‘const Trinity::ChatCommands::Variant<Trinity::ChatCommands::Hyperlink<Trinity::Hyperlinks::LinkTags::gameevent>, unsigned int>’ and ‘std::vector<GameEventData>::size_type’ {aka ‘long unsigned int’})
```

Reply:
what is the tag base for the `gameevent` link tag? can you `static_cast` it to `uint32`?